otwcode/otwarchive
# AO3-6623  Paginate works in series

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6623

## Purpose

This PR updates the view rendered by the `SeriesController`'s `show` action, to paginate the respective model's works.

## Testing Instructions

- Go to series page with > 25 works
- Confirm that works are paginated

**Screenshots:**
<details>
<summary>Series page with paginated works, page 1</summary>
<br>
<img src="https://github.com/otwcode/otwarchive/assets/6893300/3bd73c76-f21a-4132-be6f-6c2f8e910d63"/>
</details>

<details>
<summary>Series page with paginated works, page 2</summary>
<br>
<img src="https://github.com/otwcode/otwarchive/assets/6893300/0952e15c-e1f2-4326-9e6d-d44984a1d229"/>

</details>

<details>
<summary>Series page with works, no pagination</summary>
<br>
<img src="https://github.com/otwcode/otwarchive/assets/6893300/3230a137-2f42-4620-9ea5-fdf97ce0d2d8"/>

</details>

<details>
<summary>Series page with no works</summary>
<img src="https://github.com/otwcode/otwarchive/assets/6893300/0ab2e1b9-4bec-47c0-96da-7c57a69c6f18"/>

</details>

## Credit

Brandon Walsh, they/them/he/him
